multik-api/src/main/kotlin/org/jetbrains/kotlinx/multik/ndarray/operations/Add.kt

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.kotlinx.multik.ndarray.operations
+
+import org.jetbrains.kotlinx.multik.ndarray.data.*
+
+public fun <T : Number> D1Array<T>.append(n: D1Array<T>): D1Array<T>{
+    val totalSize :Int = this.size+n.size
+    when (this.dtype) {
+        DataType.IntDataType -> {
+            var dest = IntArray(totalSize)
+            System.arraycopy(this.data.getIntArray(),0,dest,0,this.size)
+            System.arraycopy(n.data.getIntArray(), 0, dest,this.size, n.size)
+            var appendedArr:D1Array<T> = D1Array(data = initMemoryView<T>(totalSize, this.dtype){dest[it] as T},dim=D1, shape = intArrayOf(totalSize), dtype = this.dtype)
+            return appendedArr
```

Comment:
You have left out the behavior on view arrays.
For example, if I do this:
```kotlin
ndarray1[1..3].append(ndarray2)
```
The result will be incorrect

Reply:
Hey @devcrocod , i am unable to understand what type of use case is this, can you provide me with some code snippet other than this?

Reply:
Ok.
Sorry for taking so long to answer
Here is a small example, in fact this is the top example, but it can be reproduced:
```kotlin
val a = mk.ndarray(mk[1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
val b = mk.ndarray(mk[2, 3])

println(a[2..5]) // [3, 4, 5]

println(a[2..5].append(b))
/*
expected: [3, 4, 5, 2, 3]
actual: [1, 2, 3, 2, 3]
*/
```